Helm: Add basic documentation about native histograms (#4530)
grafana/mimir: docs/sources/helm-charts/mimir-distributed/configure/_index.md

---

> description: This section provides links to Grafana Mimir Helm chart configuration topics.

Reviewer: ```suggestion
description: Learn how to configure Grafana Mimir or GEM via the Helm chart.
```
